social_protection/services.py

```diff
@@ -59,6 +59,29 @@ def update(self, obj_data):
     def delete(self, obj_data):
         return super().delete(obj_data)
 
+    def _data_for_json_ext(self, obj_data):
+        beneficiary = Beneficiary.objects.get(id=obj_data.get("id"))
+        individual = beneficiary.individual
+        benefit_plan = beneficiary.benefit_plan
+        individual_identity_string = f'{individual.first_name} {individual.last_name}'
+        json_ext_data = {"individual_identity": individual_identity_string,
+                         "benefit_plan_string": benefit_plan.__str__()}
+        return json_ext_data
+
+    def _data_for_json_ext_create(self, obj_data):
+        individual = Individual.objects.get(id=obj_data.get("individual_id"))
+        benefit_plan = BenefitPlan.objects.get(id=obj_data.get("individual_id"))
+        individual_identity_string = f'{individual.first_name} {individual.last_name}'
+        json_ext_data = {"individual_identity": individual_identity_string,
```

Comment:
Same as above. Moreover what is the difference between those two methods you created?

Can we maybe name those method in better way? Seems quite the same and at the first blink of eye is not clear for me what are the purposes of methods you implemented.

Reply:
it overrides service methods